packages/cozy-procedures/src/containers/DocumentsDataForm.jsx

```diff
@@ -0,0 +1,21 @@
+import { connect } from 'react-redux'
+import context from '../redux/context'
+import { fetchDocument, getDocuments } from '../redux/documentsDataSlice'
+
+const mapStateToProps = state => ({
+  data: getDocuments(state)
+})
+
+const mapDispatchToProps = {
+  fetchDocument
+}
+
+const DocumentsDataFormContainer = Component =>
```

Comment:
Can we keep the same naming convention as the other containers?

Reply:
Since I'll have to edit other containers... Can I rename all the Container* with a suffix ? :D

Reply:
Yeah sure, as long as there's only one naming convention I'm happy :)

Reply:
We'll work on that in a future PR